src/dev/storybook/aliases.ts

```diff
@@ -28,6 +28,7 @@ export const storybookAliases = {
   dashboard: 'src/plugins/dashboard/.storybook',
   data: 'src/plugins/data/.storybook',
   discover: 'src/plugins/discover/.storybook',
+  discover_log_explorer: 'x-pack/plugins/discover_log_explorer/.storybook',
```

Comment:
Could we also add it to `.buildkite/scripts/steps/storybooks/build_and_upload.ts` as recently documented in https://github.com/elastic/kibana/pull/160473?